org.osate.ge/src/org/osate/ge/internal/diagram/runtime/DiagramSerialization.java

```diff
@@ -126,16 +149,19 @@ private static long getMaxId(final org.osate.ge.diagram.DiagramElement de) {
 		return max;
 	}
 
-	private static void readElements(final DiagramModification m, final DiagramNode container,
-			final org.osate.ge.diagram.DiagramNode mmContainer, final Set<Long> usedIdSet) {
+	private static void readElements(final DiagramModification m, final ContentFilterProvider contentFilterProvider,
+			final DiagramNode container, final org.osate.ge.diagram.DiagramNode mmContainer,
+			final Set<Long> usedIdSet, final boolean usingLegacyContentFilters) {
 		for (final org.osate.ge.diagram.DiagramElement mmElement : mmContainer.getElement()) {
-			createElement(m, container, mmContainer, mmElement, usedIdSet);
+			createElement(m, contentFilterProvider, container, mmContainer, mmElement, usedIdSet,
+					usingLegacyContentFilters);
 		}
 	}
 
-	private static void createElement(final DiagramModification m, final DiagramNode container,
-			final org.osate.ge.diagram.DiagramNode mmContainer, final org.osate.ge.diagram.DiagramElement mmChild,
-			final Set<Long> usedIdSet) {
+	private static void createElement(final DiagramModification m, final ContentFilterProvider contentFilterProvider,
```

Comment:
mmContainer does not seem to be used in createElement.

Reply:
Pushed a fix. Not sure how long that Good catch.